harpy/_validations.py

```
@@ -293,22 +295,22 @@
-def validate_regions(regioninput, genome):
+def validate_regions(regioninput: int | str, genome: str) -> str:
     """validates the --regions input of harpy snp to infer whether it's an integer, region, or file"""
     try:
         # is an int
         region = int(regioninput)
+        if region < 10:
+            print_error("window size too small", "Input for [green bold]--regions[/green bold] was interpreted as an integer to create equal windows to call variants. Integer input for [green bold]--regions[/green bold] must be greater than or equal to [green bold]10[/green bold].")
+            sys.exit(1)
+        else:
+            return "windows"
     except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)